web/src/styles/components/calendar.scss

```diff
@@ -15,7 +15,7 @@
   justify-content: space-between;
   align-items: left;
   padding: 1em 1.75em;
-  border: 1px solid $base-lighter;
+  border: 1px solid $primary-light;
```

Comment:
It seems like this will change the border color for ALL the tax prompts (including the Pending message, for example)

Is that what the designers want? Or did they only want to change the Get Access prompt?

Reply:
Nice chat, Anne. Thanks!